### PR TITLE
Create Aragon dao addresses Models - Ready for review

### PR DESCRIPTION
**Pull request by henrystats**
This pr creates a new Aragon model for dao addresses, I'll be separating out all the dao_creator_tools so they're under individual project files. 

Starting with Aragon as the Aragon team wants to add a new column to the addresses_model for Aragon.

### CODE REVIEW

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @henrystats 

This PR adds aragon dao addresses spells, which consists of [`app_dao_addresses`, `client_dao_addresses`] on [`ethereum`, `gnosis`, `polygon`] chains.

File: models/aragon/gnosis/aragon_gnosis_dao_addresses.sql [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{{config(alias='dao_addresses')}}
+
+{% set aragon_models = [
+ref('aragon_polygon_client_dao_addresses')
```
Comment (jeff-dude) [inline models/aragon/gnosis/aragon_gnosis_dao_addresses.sql:4]: assume this should be `gnosis`?

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: models/aragon/gnosis/aragon_gnosis_dao_addresses.sql (hunk 1 above)
Comment (henrystats) [reply models/aragon/gnosis/aragon_gnosis_dao_addresses.sql]: yes :( updated now ser ty

**Review by jeff-dude — APPROVED** [review]
thank you

**Pull request merged**